docker/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN git clone https://github.com/pytorch/pytorch
 COPY . /pytorch/xla
 RUN if [ "${cloud_build}" = true ]; then github_branch="${release_version}" && \
   if [ "${release_version}" = "nightly" ]; then github_branch="master"; fi && \
-  cd /pytorch && rm -rf xla && git clone -b "${github_branch}" --recursive https://github.com/pytorch/xla; fi
+  cd /pytorch && rm -rf xla && git clone -b "${github_branch}" --recursive https://github.com/pytorch/xla && \
+  cd / && git clone -b "${github_branch}" --recursive https://github.com/pytorch-tpu/examples tpu-examples; fi
```

Comment:
Can we maybe format this such that every command is on one line?
```
  cmd1 && \
    cmd2 && \
    cmd3 && \
    ...
```

Reply:
how does it look now?